_posts/2021-11-19-hhvm-4.136.markdown

```diff
@@ -0,0 +1,24 @@
+---
+title: "HHVM 4.136"
+layout: post
+author: atry
+category: blog
+---
+
+HHVM 4.136 is released! HHVM 4.131&ndash;4.135 remain supported, as do the 4.102 and 4.128 LTS releases.
+
+# Highlights
+
+- Add IDE quickfix for missing interface methods.
+- [The `readonly` feature](https://docs.hhvm.com/hack/readonly/introduction) is no longer considered unstable.
+  - The `readonly` modifier is now also visible to the reflection API via the `isReadonly()` or `returnsReadonly()` functions.
+- An experimental option `hhvm.jit_attempt_super_inlining` is added to `.hhconfig`, which enables a novel strategy in HHVM attempting to const fold nearly-const values.
```

Comment:
For changes like this, there's a few frequent follow-up questions that are good to answer in the release notes if possible:

- Why would I want to use it for my deployment?
- Why would I *not* want to use it?
- Does FB encourage/discourage people using it?
- If FB encourages it, why is it not enabled by default?

The goal here is to make the entry something that people can directly act on (or decided not to)

If we don't have solid answers, it might be better to delay mentioning this for a future release